docs/product/issues/issue-details/sentry-ai/index.mdx

```diff
@@ -0,0 +1,68 @@
+You can find it in the Solutions Hub in the sidebar of the issue details page for any error:
+
+![Setup](./img/setup.png)
+
+Once you've set up Sentry AI, you'll be able to use the following features:
+
+- [**Issue Summary**](#issue-summary): A quick summary of all the data in the issue to help you understand what's going on at a glance.
+- [**Autofix**](#autofix): A collaborative workflow to find the root cause of an error and create a PR that fixes it.
+
+## Issue Summary
+
+Issue Summary quickly orients you with the issue by summarizing key insights across event and issue level metadata.
+
+When you open an issue, you'll see a quick overview of what's going wrong in the issue in the Solutions Hub in the sidebar:
+
+![Solutions Hub](./img/solutions-hub.png)
+
+Opening the Solutions Hub will show you a more detailed overview of the issue. It will summarize what's going wrong, a possible cause, and if relevant, insights from trace-connected issues:
+
+![Issue Summary](./img/issue-summary.png)
+
+## Autofix
+
+_Note: Autofix replaces Suggested Fix, which was an earlier experiment that we have sunset._
+
+When you want to understand the root cause more deeply, you can use Autofix. It uses the issue details, your codebase (integrated through GitHub), and its interactions with you to gain context and reason through the problem.
+
+When you start an Autofix run, you'll see the following screen, with an option to share initial helpful context on the issue:
+
+![Autofix Start Screen](./img/autofix-start-screen.png)
+
+Autofix then starts analyzing the issue and the relevant code. You can see its status at the bottom of the screen. As it goes through the process, Autofix will share its key insights along the way. You can expand each one to see the exact event data and code that it used to justify it. These insights build up a chain of reasoning to the root cause of the error.
+
+Autofix won't know everything, so the best results come through collaboration. At any point, you can jump in with helpful context or feedback. You can also directly edit its line of reasoning. Just click the "Edit" button above any line, provide any thoughts or instructions you have, and Autofix will rethink from that point. Autofix can also recognize when it doesn't know something, and ask you for the context it needs to continue.
+
+![Autofix Insights](./img/autofix-qna.png)
+
+Once you reach a root cause you're happy with, or have enough context to propose your own, Autofix will use the same process to find a fix. When complete, you'll see the diff of its proposed code changes. You can directly reject, edit, rethink, and give feedback on the changes. You can also ask Autofix to add tests to ensure the issue doesn't happen again.
+
+![Autofix Results](./img/autofix-results.png)
+
+Once you're happy with the changes, Autofix can create a draft PR for you to review and merge in GitHub.
```

Comment:
```suggestion

<Note>

 Autofix replaces Suggested Fix (an earlier feature that has been sunset).
 
 </Note>

**Autofix** uses the issue details, your codebase (integrated through GitHub), and its interactions with you to gain context, reason through the problem, and get a deeper understanding of the root cause. 

When you start an Autofix run, you'll see the following screen, with an option to share any additional context:

![Autofix Start Screen](./img/autofix-start-screen.png)

Autofix will then analyze the issue and relevant code to help identify the root cause. You'll see the progress displayed at the bottom of the screen. Autofix will share key insights as they come up and you'll have the option to expand each. You'll see the event data and code that support Autofix conclusions, building a clear chain of reasoning back to the root cause.

Since Autofix doesn’t have all the context, collaboration leads to the best results. You can step in at any time to add context or feedback. To edit its reasoning, click the "Edit" button above any line to provide instructions or corrections, and Autofix will adjust its approach accordingly. If Autofix encounters gaps in its understanding, it will ask for your input.

![Autofix Insights](./img/autofix-qna.png)

When you’ve identified the root cause, or gathered enough information to propose your own solution, Autofix will use the same process to suggest a fix. It will present a diff of the proposed code changes. You can then:

- Accept, reject, or edit the changes.
- Provide feedback or ask Autofix to rethink its approach.
- Request tests to ensure the issue doesn’t recur.

![Autofix Results](./img/autofix-results.png)

Once you're happy with the changes, Autofix can create a draft PR for you to review and merge in GitHub.
```